lang/python/python3-importlib-metadata/Makefile

```diff
@@ -0,0 +1,38 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=importlib_metadata
```

Comment:
This is so wrong. It should have the same name as the folder, where is the Makefile.